gtk4/src/subclass/selection_model.rs

```diff
@@ -0,0 +1,173 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::Bitset;
+use crate::SelectionModel;
+use glib::subclass::prelude::*;
+use glib::translate::*;
+use glib::Cast;
+
+pub trait SelectionModelImpl: ObjectImpl {
+    fn get_selection_in_range(&self, model: &Self::Type, position: u32, n_items: u32) -> Bitset;
```

Comment:
This one has a default impl, as have many others. See https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/-/blob/8549046994146118850bae111bf1c7e942d55a1b/gstreamer/src/subclass/child_proxy.rs#L10-26 how to handle that

Reply:
Other functions here also have default implementations, you only added it for this one (but that looks correct) :)